### PR TITLE
updated event tutorial for stax

### PR DESCRIPTION
**Pull request by rohitner**
added some images

### CODE REVIEW

File: _tutorials/event/stax.md [hunk 1]
```diff
@@ -3,54 +3,23 @@ layout: post
...
-|Bread Board                            | 1|
-|Power supply for Arduino-5V            | 1|
-|Power supply for Motor-as specified    | 1|
-
 #### Line Following
 
 ##### 1.IR LED Pair
-**Working**- IR sensor or an Infrared sensor consist an IR LED coupled with a Photodiode. This pair is generally called an IR pair or Photocoupler. It works on the principle that an infrared sensor emits Infrared radiations and the Photodiode senses the same. Photodiode resistance changes according to the amount of IR radiation falling on it, hence changing the voltage drop against it. Further, by using the voltage comparator (like LM358) we can sense the voltage change and generate the output accordingly.
+**Working**- IR sensor or an Infrared sensor consists an IR LED coupled with a Photodiode. This pair is generally called an IR pair or Photocoupler. An infrared sensor emits Infrared radiations and the Photodiode senses the same. Photodiode resistance changes according to the amount of IR radiation falling on it, hence changing the voltage drop against it. Further, by using the voltage comparator (like LM358) we can sense the voltage change and generate the output accordingly.
```
Comment (narayanaditya95) [inline _tutorials/event/stax.md]: `consists of`?

Comment (narayanaditya95) [reply _tutorials/event/stax.md]: No reason why photodiode and infrared must begin with capital letters.

File: _tutorials/event/stax.md [hunk 2]
```diff
@@ -3,54 +3,23 @@ layout: post
...
-|Color sensor module TCS3200            | 1 |
-|Pair of /IR Sensor Array               | 1 |
-|DC Motors                              | 2 |
-|Wheels                                 | 2 |
-|Castor Wheel                           | 1 |
-|Jumper wires                           | As required |
-|Bread Board                            | 1|
-|Power supply for Arduino-5V            | 1|
-|Power supply for Motor-as specified    | 1|
-
```
Comment (narayanaditya95) [inline _tutorials/event/stax.md]: Why are we removing this?

**Comment by rohitner** [comment]
-

**Pull request merged**